base/resources/scripts/languagecheck.js

```diff
@@ -0,0 +1,40 @@
+/* Nodejs to set up a server on 8081 to modify the argv.json file
+   used by in-browser VSCode. Uses the accept-language header.
+   Also used by jupyter to determine language.
+ */
+var http = require("http");
+var fs = require("fs");
+
+http
+  .createServer(function (req, res) {
+    res.writeHead(200, { "Content-Type": "text/html" });
```

Comment:
* We don't know the operation will succeed.
* I would say your response looks more like `text/plain`.

Reply:
The bullet point "We don't know the operation will succeed" is the only piece of feedback I don't really understand.

Reply:
You were writing out a 200 (success) response code right out of the gate, but we don't know, at that point, that the task at hand isn't going to fail. I think you covered this in your latest commits by writing out a 500 for failure cases.